samples/Tests/AdaptiveCard.Version1.1.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.1",
```

Comment:
Because your changing some test cards and adding new ones, you need to run the UWP TestApp to generate new payload to compare against.